app/controllers/catalog_controller.rb

```diff
@@ -379,6 +379,7 @@ def raw
     config.add_restrictions_field 'userestrict_tesim', label: 'Use & Permissions', helper_method: :convert_rights_urls
 
     # Collection Show Page - Background Section
+    config.add_background_field 'para_tesim', label: '', helper_method: :render_html_tags
```

Comment:
@gkostin1966 this renders as an empty `<dt>` --- and since we're using a `<dl>` here that can't be empty.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/537867/212372787-02ea7ad1-401c-4336-9bdf-6b188c013448.png">

Reply:
@cmkpowell would "Description" work?

Reply:
I think Note would be the safest term.

Reply:
@respinos I think @estelendur has been assigned this ticket.  My name is one here because I moved it from the old mlibrary/arclight repository.

Reply:
@gkostin1966 🤣 DUH. Forgot about that.

Reply:
Changed the strategy in a way that renders this irrelevant